### DISCUSSION TITLE
Advanced Networking blocked off

### DISCUSSION THREAD

**Discussion by ToasterDEV** [LSPosed/MagiskOnWSALocal#157] [Q&A] [answered]
Hello!

Happy user of this toolkit, just with a slight question regarding its differences from regular WSA.

I have attempted to use the Advanced Networking feature to be able to use Google Home within my network, but for some reason it appears that something is using a VPN within the installation, even though I have not configured a provider of such nature within the VM.

![image](https://user-images.githubusercontent.com/5206262/193379116-efe36044-22e8-440c-80da-ae8bcac34306.png)
![image](https://user-images.githubusercontent.com/5206262/193379119-060e4543-0ff2-44a3-83cc-11f936b30cc0.png)

Is this more of a WSA issue, or is it because of the way it's being deployed?

Thanks for the help!

**Comment by Howard20181** [comment, marked as answer]
I use the same way to deploy WSA and I'm able to enable Advance Network. Maybe a software in Windows are using a tunnel.

**Reply by ToasterDEV** [reply]
Would something akin to Mullvad or Wireguard, or perhaps the VMWare adapters be able to create an interference, even if currently disabled and/or not running when the error pops up?

**Reply by Howard20181** [reply]
Tested that VMWare adapters does not cause the problem. Wintun does.

**Reply by ToasterDEV** [reply]
Hello, sorry for necro-ing the thread but found a solution to have both Wireguard and WSA cooperate. It seems we just need to add the following IPs to the AllowList: `::/128, 0.0.0.0/1, 128.0.0.0/1` according to this [post](https://old.reddit.com/r/WSA/comments/yaetta/2_questions_start_wsa_silently_and_wireguard_not/) I found.

Thanks for the help!
